src/libs/modals/modals/ModalConfirm/ModalConfirm.tsx

```diff
@@ -45,7 +47,12 @@ export const ModalConfirm: ModalFC<ModalCreateConfirmData> = ({
   }, [context, eventData]);
 
   return (
-    <Modal id={id} title="Confirm Transaction" size={'md'}>
+    <Modal
+      onClose={() => onClose && onClose()}
```

Comment:
why not just 

`onClose={onClose}`

Reply:
Actually the onClose callback is not in use right now so I'm removing it.
I added it in the beginning because the confirm modal had effect on the status of the button but not anymore.